app/lib/actions/katello/content_view_puppet_environment/clone.rb

```diff
@@ -32,7 +32,6 @@ def plan(from_version, options)
 
               concurrence do
                 plan_action(Katello::Repository::MetadataGenerate, clone) if environment
-                plan_action(ElasticSearch::ContentViewPuppetEnvironment::IndexContent, id: clone.id)
```

Comment:
This needs to stay so that the puppet modules are still indexed properly (I will be updating this in my puppet module conversion).